client/src/app/site/pages/meetings/pages/motions/pages/motion-detail/components/motion-detail-view/motion-detail-view.component.scss

```
@@ -158,4 +158,20 @@ span {
 
 .motion-detail-nav-button {
     min-width: 110px;
+    max-width: 20vw;
+
+    span {
+        white-space: nowrap !important;
+        text-overflow: ellipsis;
+        overflow: hidden;
+
+        @supports (-webkit-line-clamp: 3) {
+            overflow: hidden;
+            text-overflow: ellipsis;
```

Remove duplicate code.
```suggestion
        text-overflow: ellipsis;
        overflow: hidden;

        @supports (-webkit-line-clamp: 3) {
```

Also isn't there an alternative to using webkit specific features?

---

AFAIK if I want to do a multi-line ellipsed text via css, I need to use webkit.

Leaving css behind, it would maybe be possible to use JavaScript to read out/calculate the maximum dimensions of this width-variable element, calculate the maximum number of symbols that will fit within 3 lines on that width, manually ellipse the text in question with the hope that the calculations were correct, and then re-calculate all that whenever the viewport-width changes.

I judged the latter to neither be elegant nor worth the effort for two small nav-buttons.

---

I will remove the duplicate code though